From 4c3c67403a60c0bf292349ddc1eaf1daadb07b99 Mon Sep 17 00:00:00 2001
From: Alexander Graf <agraf@suse.de>
Date: Wed, 3 Jul 2013 14:51:11 +0200
Subject: [PATCH] usb: phy: Fix phy-samsung-usb when built as module
Patch-Mainline: Not yet

When building phy-samsung-usb, I get the following on module load:

  phy_samsung_usb: module license 'unspecified' taints kernel.
  phy_samsung_usb: Unknown symbol clk_get_rate (err 0)

Add the respective module meta information to the file that got
removed in dc2377d, so that the module can identify itself as GPL
again.

CC: Vivek Gautam <gautam.vivek@samsung.com>
CC: Kukjin Kim <kgene.kim@samsung.com>
CC: Felipe Balbi <balbi@ti.com>
Signed-off-by: Alexander Graf <agraf@suse.de>
---
 drivers/usb/phy/phy-samsung-usb.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/usb/phy/phy-samsung-usb.c b/drivers/usb/phy/phy-samsung-usb.c
index 7b118ee5..2e03f3a 100644
--- a/drivers/usb/phy/phy-samsung-usb.c
+++ b/drivers/usb/phy/phy-samsung-usb.c
@@ -234,3 +234,7 @@ int samsung_usbphy_get_refclk_freq(struct samsung_usbphy *sphy)
 	return refclk_freq;
 }
 EXPORT_SYMBOL_GPL(samsung_usbphy_get_refclk_freq);
+
+MODULE_DESCRIPTION("Samsung USB phy controller helpers");
+MODULE_AUTHOR("Praveen Paneri <p.paneri@samsung.com>");
+MODULE_LICENSE("GPL");
-- 
1.7.10.4

